_data/software/perl.yml

```diff
@@ -0,0 +1,36 @@
+---
+schemaVersion: '1.0'
+description:
+  whatItDoes:
+    en: >-
+      Perl is a highly capable, feature-rich programming language with over 30
+      years of development.
+    fr: >-
+      Perl est un langage de programmation très performant et riche en
+      fonctionnalités avec plus de 30 ans de développement.
+category: '43232400'
+homepageURL:
+  en: 'https://www.perl.org/'
+  fr: 'https://www.perl.org/'
+licences:
+  - URL:
+      en: 'https://dev.perl.org/licenses/'
+      fr: 'https://dev.perl.org/licenses/'
+    spdxID: GPL-2.0-or-later
```

Comment:
It looks like it's GPL 1 or later `GPL-1.0-or-later`

Reply:
It is, but neither the OSI or FSF endorse GPL-1.0-or-later, which would indicate that it falls out of what we can accept. As it's a "1.0-or-later" we can accept it as 2.0-or-later. 

Not sure how we'd want to note this, or just to keep it as info on hand.